src/Extensions/ContentReviewCMSExtension.php

```diff
@@ -17,6 +18,8 @@
 /**
  * CMSPageEditController extension to receive the additional action button from
  * SiteTreeContentReview::updateCMSActions()
+ *
+ * @extends LeftAndMainExtension<CMSMain>
```

Comment:
```suggestion
 * @extends LeftAndMainExtension<CMSPageEditController|ContentReviewCMSExtension>
```

Reply:
As discussed in person, both `CMSPageEditController` and `CMSPageSettingsController` which this is applied to are subclasses of `CMSMain`, so it makes sense to use that superclass as the typehint here.